### PR TITLE
Add contributors to changelog.md

### PR DESCRIPTION
**Pull request by jasonblais**

### CODE REVIEW

File: source/administration/changelog.md [hunk 1]
```diff
@@ -200,6 +200,44 @@ The following config settings will only work on servers with an Enterprise Licen
 - Webhook attachments don’t show up in search results.
 - Count of unread mentions are sometimes mixed when switching between multiple teams.
 
+### Contributors
+
+Many thanks to all our contributors. In alphabetical order:
+
+/platform
+
+[asaadmahmoodspin](https://github.com/asaadmahmoodspin), [coreyhulen](https://github.com/coreyhulen), [crspeller](https://github.com/crspeller), [DavidLu1997](https://github.com/DavidLu1997), [eadmund](https://github.com/eadmund), [enahum](https://github.com/enahum), [esethna](https://github.com/esethna), [hmhealey](https://github.com/hmhealey), [jasonblais](https://github.com/jasonblais), [jwilander](https://github.com/jwilander), [lfbrock](https://github.com/lfbrock), [maruTA-bis5](https://github.com/maruTA-bis5), [Rudloff](https://github.com/Rudloff), [samogot](https://github.com/samogot), [yuters](https://github.com/yuters)
```
Comment (lindy65) [inline source/administration/changelog.md]: I think Asaad might have changed his GitHub username as his GitHub link is giving a `404 page not found` error
[asaadmahmoodspin](https://github.com/asaadmahmoodspin)

File: source/administration/changelog.md [hunk 2]
```diff
@@ -200,6 +200,44 @@ The following config settings will only work on servers with an Enterprise Licen
...
+
+/desktop
+
+[jasonblais](https://github.com/jasonblais), [jnugh](https://github.com/jnugh), [Razzeee](https://github.com/Razzeee), [timroes](https://github.com/timroes), [yuya-oc](https://github.com/yuya-oc)
+
+/android
+
+[coreyhulen](https://github.com/coreyhulen), [lfbrock](https://github.com/lfbrock)
+
+/ios
```
Comment (lindy65) [inline source/administration/changelog.md]: `ios` > `iOS` ??

Should we capitalize the names of `Android`, etc?

File: source/administration/changelog.md [hunk 3]
```diff
@@ -200,6 +200,44 @@ The following config settings will only work on servers with an Enterprise Licen
...
+
+[coreyhulen](https://github.com/coreyhulen), [hmhealey](https://github.com/hmhealey), [lfbrock](https://github.com/lfbrock)
+
+/docs
+
+[asaadmahmoodspin](https://github.com/asaadmahmoodspin), [esethna](https://github.com/esethna), [it33](https://github.com/it33), [jasonblais](https://github.com/jasonblais), [lfbrock](https://github.com/lfbrock), [lindy65](https://github.com/lindy65)
```
Comment (lindy65) [inline source/administration/changelog.md]: I think Asaad might have changed his GitHub username as his GitHub link is giving a `404 page not found` error
[asaadmahmoodspin](https://github.com/asaadmahmoodspin)

File: source/administration/changelog.md [hunk 4]
```diff
@@ -200,6 +200,44 @@ The following config settings will only work on servers with an Enterprise Licen
...
+
+/mattermost-docker
+
+[npcode](https://github.com/npcode)
+
+/mattermost-driver-javascript
```
Comment (lindy65) [inline source/administration/changelog.md]: `javascript` > `JavaScript` ??

File: source/administration/changelog.md (hunk 2 above)
Comment (jasonblais) [inline source/administration/changelog.md]: this is actually the [/ios](https://github.com/mattermost/ios/) repo, which isn't capitalized

File: source/administration/changelog.md (hunk 4 above)
Comment (jasonblais) [inline source/administration/changelog.md]: same here, the repo name is lowercase

**Comment by jasonblais** [comment]
Thanks! Updated

File: source/administration/changelog.md (hunk 2 above)
Comment (lindy65) [inline source/administration/changelog.md]: Ah thanks!  Didn't know that :)

File: source/administration/changelog.md (hunk 4 above)
Comment (lindy65) [inline source/administration/changelog.md]: Cool, thanks!

**Comment by lindy65** [comment]
+1

**Comment by esethna** [comment]
+1

**Pull request merged**